api/pagination.go

```diff
@@ -29,6 +29,11 @@ func addPaginationHeaders(w http.ResponseWriter, r *http.Request, p *models.Pagi
 		url.RawQuery = query.Encode()
 		header += "<" + url.String() + ">; rel=\"next\", "
 	}
+	if p.Page > 1 {
```

Comment:
To confirm: page numbers start at 1 and not 0, right?

Reply:
yup! the page is part of query params even if the site has less than 15 identity users 
`/users?page=1&per_page=15&sort=created_at`